draft-ietf-dnsop-svcb-https.md

```diff
@@ -573,7 +573,9 @@ destinations with a proxy that does not provide SVCB query capability
 (e.g. through an affiliated DNS resolver), the client would have to perform
 SVCB queries though a separate resolver.  This might disclose the client's
 destinations to an additional party, creating privacy concerns.  If these
-concerns apply, the client SHOULD disable SVCB resolution.
+concerns apply, the client SHOULD mitigate these concerns in some way,
```

Comment:
```suggestion
concerns apply, SVCB-optional clients SHOULD disable SVCB resolution.
SVCB-reliant clients cannot function in this configuration, so they must
arrange for SVCB resolution with appropriate privacy and routing properties.
```

Reply:
Perhaps we also allow for "SVCB-optional clients SHOULD disable SVCB resolution
unless they arrange for SVCB resolution with appropriate privacy and routing properties."